common/src/main/java/org/apache/celeborn/common/network/client/TransportClient.java

```diff
@@ -228,7 +228,7 @@ public void onFailure(Throwable e) {
     try {
       return result.get(timeoutMs, TimeUnit.MILLISECONDS);
     } catch (Exception e) {
-      throw new IOException("Exception in sendRpcSync", e);
+      throw new IOException("Exception in sendRpcSync to:" + this.getSocketAddress(), e);
```

Comment:
Here only called when PartitionReader fetch open chunk, if you want to do this, maybe we should also handle `askSync`

Reply:
This is used for flink write process. We wanted to know which worker client failed to push.